### PR TITLE
MOYO header

### PR DESCRIPTION
**Pull request by DanielYurchenco**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,10 +5,37 @@
...
+          rel="stylesheet">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet"
+          href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="https://www.google.com.ua/?hl=ru" class="logo">
```
Comment (Viktor-Kost) [inline src/index.html:20]:
```suggestion
      <a href="#" class="logo">
```

File: src/index.html [hunk 2]
```diff
@@ -5,10 +5,37 @@
...
+        <img src="images/logo.png" alt="MOYO logo">
+      </a>
+      <nav class="nav">
+        <ul class="navbar">
```
Comment (Viktor-Kost) [inline src/index.html:24]:
```suggestion
        <ul class="nav__list">
```

File: readme.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Moyo header
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://DanielYurchenco.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://DanielYurchenco.github.io/layout_moyo-header/report/html_report/)
```
Comment (Viktor-Kost) [inline readme.md:4]: please add these links to a PR description

File: src/index.html [hunk 4]
```diff
@@ -5,10 +5,37 @@
...
+          <li class="navlist"><a href="https://www.google.com.ua/?hl=ru" class="navlink apple is-active">APPLE</a></li>
+          <li class="navlist"><a href="https://www.google.com.ua/?hl=ru" class="navlink sumsung ">SUMSUNG</a></li>
+          <li class="navlist"><a href="https://www.google.com.ua/?hl=ru" class="navlink smartphone ">SMARTPHONES</a></li>
+          <li class="navlist">
+            <a href="https://www.google.com.ua/?hl=ru"
+               class="navlink lap_com"
+               data-qa="hover">LAPTOPS & COMPUTERS</a>
+          </li>
```
Comment (Viktor-Kost) [inline src/index.html:32]: 1. apply code styling recommendations from a checklist to other items as well
2. use BEM methodology for naming classes
3. no needs to set a link. You can just use `#`
4. use lowercase at html and change this value at css
```suggestion
          <li class="nav__item">
            <a 
              href="#laptops&computers"
              class="nav__link"
              data-qa="hover"
            >
              laptops & computers
            </a>
          </li>
```

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,131 @@
...
+    transition-duration: 0.01ms !important;
+    scroll-behavior: auto !important;
+  }
+}
+/* main style header navigation */
+
+.header {
+  display: flex;
+  align-items: center;
+  margin: 10px 50px;
```
Comment (Viktor-Kost) [inline src/style.css:74]:
```suggestion
  padding: 0 50px;
```

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,131 @@
...
+}
+
+.navbar {
+  display: flex;
+}
+
+.navlist {
```
Comment (Viktor-Kost) [inline src/style.css:81]: first item shouldn't have margin. use `:not(:first-child)`

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,131 @@
...
+  margin-left: 20px;
+}
+
+.navlink {
+  text-decoration: none;
+  color: black;
```
Comment (Viktor-Kost) [inline src/style.css:87]: apply `hex` instead of reserved color words

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,131 @@
...
+  position: relative;
+  padding-top: 15px;
+  padding-bottom: 15px;
```
Comment (Viktor-Kost) [inline src/style.css:90]: links should have a bigger padding value
```suggestion
  padding: 23px 0;
```

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,131 @@
...
+.navlink:hover::after {
+  color: #00acdc;
+  content: "";
+  width: 100%;
+  height: 3px;
+  border-radius: 5px;
+  background-color: #00acdc;
+  position: absolute;
+  right: 0;
+  bottom: -1px;
```
Comment (Viktor-Kost) [inline src/style.css:102]:
```suggestion
  bottom: 0;
```

File: src/style.css [hunk 10]
```diff
@@ -1,3 +1,131 @@
...
+  position: relative;
+  padding-top: 15px;
+  padding-bottom: 15px;
+}
+
+.navlink:hover::after {
+  color: #00acdc;
+  content: "";
+  width: 100%;
+  height: 3px;
```
Comment (Viktor-Kost) [inline src/style.css:97]:
```suggestion
  height: 4px;
```

File: src/style.css [hunk 11]
```diff
@@ -1,3 +1,131 @@
...
+  border-radius: 5px;
+  background-color: #00acdc;
```
Comment (Viktor-Kost) [inline src/style.css:99]: move repeated colors to variables

File: src/style.css [hunk 12]
```diff
@@ -1,3 +1,131 @@
...
+  position: absolute;
+  right: 0;
+  bottom: -1px;
+}
+
+.navlink:focus {
+  color: #00acdc;
+}
+
+.nav {
```
Comment (Viktor-Kost) [inline src/style.css:109]: you can just make your header as `flex` and use `justify-content: space-between`

File: src/style.css [hunk 13]
```diff
@@ -1,3 +1,131 @@
...
+  margin-left: auto;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::before {
```
Comment (Viktor-Kost) [inline src/style.css:117]: if you have repeated styles for a few selectors you can just use them as a list

```
.selector1, 
.selector2 {
  {{rules}}
}
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

**Comment by DanielYurchenco** [comment]
> _ No description provided. _

- [DEMO LINK](https://DanielYurchenco.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://DanielYurchenco.github.io/layout_moyo-header/report/html_report/)

File: src/index.html [hunk 14]
```diff
@@ -5,10 +5,74 @@
...
+    <link rel="stylesheet"
+          href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="images/logo.png" alt="MOYO logo">
+      </a>
+      <nav class="nav">
```
Comment (DarkMistyRoom) [inline src/index.html:23]:
```suggestion
      </a>
      
      <nav class="nav">
```

Comment (DarkMistyRoom) [reply src/index.html:23]: Add empty lines between multiline sibling blocks of HTML

File: src/index.html [hunk 15]
```diff
@@ -5,10 +5,74 @@
...
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+            href="#"
+            class="nav__link is-active">apple
+            </a>
```
Comment (DarkMistyRoom) [inline src/index.html:29]:
```suggestion
            <a
              href="#"
              class="nav__link is-active"
            >
              apple
            </a>
```

Comment (DarkMistyRoom) [reply src/index.html:29]: fix it everywhere

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
the links don't work(

File: src/index.html [hunk 16]
```diff
@@ -5,10 +5,91 @@
...
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a
+            href="#"
+            class="nav__link is-active">
+            apple
+            </a>
+          </li>
```
Comment (Viktor-Kost) [inline src/index.html:33]: apply code styling recommendations from a checklist all over the page
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="#"
              class="nav__link is-active"
            >
              apple
            </a>
          </li>
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Nice!
Minor fixes needed. 
Plus check this unresolved conversation
https://github.com/mate-academy/layout_moyo-header/pull/3925#discussion_r1350509501

File: readme.md (hunk 3 above)
Comment (volodymyr-soltys97) [inline readme.md:4]: Add these links to the pr description

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Demo link, you need to make a deploy
image

**Pull request closed without merging**